### PR TITLE
Initial instructions for QNAP

### PR DESCRIPTION
**Pull request by thopiekar**
Basically copy/pasted from openhab-qnap-qpkg/README.md

Relates to: https://github.com/openhab/openhab-docs/issues/201
Signed-off-by: Thomas Karl Pietrowski <thopiekar@gmail.com> (github: thopiekar)

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Thanks for finding the time!

File: installation/qnap.md [hunk 1]
```diff
@@ -10,4 +10,43 @@ title: QNAP NAS
...
+
+9. Access openHAB via [http://NAS_IP_or_DNS_address:8090](#)
+
+## How to uninstall
+1. Go to the "App Center" and remove the app like any other.
+2. If wanted/needed also remove "addons", "conf" and "userdata" from the your directory, eg. "openHAB2" share or "Public"/openHAB2
+  * _REMEMBER_: If you have installed openHAB2 to ".qpkg" (see "How to install", section 2.3) then all files get removed directly!
+
+## Known issues
+* Please check our issue tracker: https://github.com/openhab/openhab-distro/issues
```
Comment (BClark09) [inline installation/qnap.md]: openhab-qnap-qpkg?

**Comment by ThomDietrich** [comment]
This was not a success :-/
http://docs.openhab.org/installation/qnap.html
Will look at this later.

File: installation/qnap.md (hunk 1 above)
Comment (thopiekar) [inline installation/qnap.md]: Yes.. The project exists.. Or what is the point? 🙄

Comment (BClark09) [inline installation/qnap.md]: The link to the issue tracker goes to openhab-distro, shouldn't it have gone to the qnap-qpkg project?

**Pull request merged**